### PR TITLE
Update requirements.dev.txt

### PR DESCRIPTION
**Pull request by wouterdb**
# Description

Fix the iso6 branch to the iso6 core versions

1. @sanderr is this correct wrt to the `.dev ` in the constraint?
2. I'm not going to bump the version, as it is purely testing. 

# Self Check:

Strike through any lines that are not applicable (`~~line~~`) then check the box

- [ ] Attached issue to pull request
- [] Changelog entry
- [ ] Version number is bumped to dev version
- [x] Code is clear and sufficiently documented
- [ ] Sufficient test cases (reproduces the bug/tests the requested feature)
- [ ] Correct, in line with design
- [ ] End user documentation is included or an issue is created for end-user documentation (add ref to issue here: )

### CODE REVIEW

File: requirements.dev.txt [hunk 1]
```diff
@@ -1,3 +1,5 @@
 inmanta-dev-dependencies[module]==1.76.0; python_version <= '3.6'
 inmanta-dev-dependencies[module]==2.95.0; python_version > '3.6'
 wheel==0.42.0
+# Pin to iso 8
```
Comment (wouterdb) [inline requirements.dev.txt:4]:
```suggestion
# Pin to iso 6
```

File: requirements.dev.txt [hunk 2]
```diff
@@ -1,3 +1,5 @@
 inmanta-dev-dependencies[module]==1.76.0; python_version <= '3.6'
 inmanta-dev-dependencies[module]==2.95.0; python_version > '3.6'
 wheel==0.42.0
+# Pin to iso 6
+inmanta-core~=8.0,<8.8.dev
```
Comment (sanderr) [inline requirements.dev.txt:5]: > is this correct wrt to the .dev in the constraint?

If the goal is to have it install dev even without `--pre`, then you need the `.dev` on both.
```suggestion
inmanta-core~=8.0.dev,<8.8.dev
```

**Review by sanderr — CHANGES_REQUESTED** [review]
We could also use the constraint file from the docs? Especially if we ever branch off for iso7, because then we get the updated constraint for free. In this case it doesn't matter that much.

> I'm not going to bump the version, as it is purely testing.

The module set tests run on released versions. So unless it's a fix for those this sounds good to me.

File: requirements.dev.txt (hunk 2 above)
Comment (sanderr) [inline requirements.dev.txt:5]:
```suggestion
inmanta-core~=8.0,<8.8
```
See comment

**Review by sanderr — APPROVED** [review]

**Pull request merged**